src/api/test/functional/webui/package_controller_test.rb

```diff
@@ -284,6 +284,8 @@ def test_delete_request
     click_button 'Revoke request'
   end
 
+  # spec/features/webui/packages_spec.rb
+  # spec/controllers/webui/request_controller_spec.rb
   def test_change_devel_request
```

Comment:
Didn't think of that. Might be interesting for the rest of the team as well @openSUSE/open-build-service